addons/payment_stripe/const.py

```diff
@@ -31,3 +31,282 @@
 WEBHOOK_HANDLED_EVENTS = [
     'checkout.session.completed',
 ]
+# According to https://en.wikipedia.org/wiki/ISO_4217#Minor_unit_fractions
+CURRENCY_ISOCODE_EXPONENT = {
```

Comment:
Could you please move this definition to `payment`? The problem will be the same for all payment acquirers, not only Stripe.

Reply:
Could you also sort the currencies by name? That would make them much easier to compare and maintain.